### PR TITLE
adjust minischedules for overload offset

### PR DESCRIPTION
**Pull request by dks-mbta**

### CODE REVIEW

File: assets/src/realtime.d.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ export interface Vehicle {
   operatorFirstName: string
   operatorLastName: string
   operatorLogonTime: Date | null
+  overloadOffset?: number | undefined
```
Comment (lemald) [inline assets/src/realtime.d.ts:117]: If it's already called `overloadOffset?` I believe the question mark makes the `| undefined` redundant.

**Review by lemald — CHANGES_REQUESTED** [review]
I flagged one minor thing where a typespec is somewhat redundant.

Additionally, I think it would be good to add a few test cases in `assets/tests/components/propertiesPanel/minischedule.test.ts`. The existing tests should provide a good template, you can just take the `vehicle` defined in the test file and modify it to have an overload offset.

It would also be nice to be able to manually test this and see if it works but I know that overloads aren't very common right now.

**Comment by lemald** [comment]
> Additionally, I think it would be good to add a few test cases in assets/tests/components/propertiesPanel/minischedule.test.ts. The existing tests should provide a good template, you can just take the vehicle defined in the test file and modify it to have an overload offset.

Note though that we don't need to test all of the edge cases there, since you already have extensive unit tests for the `formattedScheduledTime` that contains most of the logic.

**Comment by dks-mbta** [comment]
Added one test, working on some more.

File: assets/tests/components/propertiesPanel/minischedule.test.tsx [hunk 2]
```diff
@@ -214,6 +214,57 @@ const vehicle: Vehicle = {
   crowding: null,
 }
 
+const vehicleWithOffset: Vehicle = {
```
Comment (lemald) [inline assets/tests/components/propertiesPanel/minischedule.test.tsx:217]: Note that if this is mostly the same as `vehicle` you can use the spread operator to write this more concisely, something like `const vehicleWithOffset = { ...vehicle, overloadOffset: 8 }`.

Comment (dks-mbta) [reply assets/tests/components/propertiesPanel/minischedule.test.tsx:301]: Yeah, I made this change in the next commit.

**Comment by dks-mbta** [comment]
@lemald The additional tests are ready for review.

**Review by lemald — APPROVED** [review]
Looks good to me!
